exercises/practice/allergies/.approaches/bitwise-and-type-alias/content.md

```diff
@@ -0,0 +1,51 @@
+# Bitwise, List and type alias
+
+```elm
+type alias AllergyAndScore = {
+  allergy: Allergy;
+  bitPosition: Int;
+}
+
+allergiesAndScores : List AllergyAndScore
+allergiesAndScores =
+    [ AllergyAndScore Eggs 1
+    , AllergyAndScore Peanuts 2
+    , AllergyAndScore Shellfish 3
+    , AllergyAndScore Strawberries 4
+    , AllergyAndScore Tomatoes 5
+    , AllergyAndScore Chocolate 6
+    , AllergyAndScore Pollen 7
+    , AllergyAndScore Cats 8
+    ]
+
+toList : Int -> List Allergy
+toList score =
+  allergiesAndScores
+    |> List.Map (\allergyAndScore -> ( 2 ^ allergyAndScore.bitPosition, allergyAndScore.allergy ))
+    |> List.filter (\( s, _ ) -> Bitwise.and s score > 0)
+    |> List.map Tuple.second
+```
+
+## In this approach
+
+This approach uses a type alias to represent the bit position and the allergy.
+
+The approach is very similar to the [Bitwise and List approach][bitwise-and-list], but uses a type alias to make the bit position explicit. It could instead be based on the [Bitwise and Dict approach][bitwise-and-dict].
```

Comment:
```suggestion
The approach is very similar to the [Bitwise and List approach][bitwise-and-list], but uses a type alias to make the bit position explicit.
It could also be based on the [Bitwise and Dict approach][bitwise-and-dict].
```